GitUI/CommandsDialogs/BrowseDialog/DashboardControl/DashboardItem.Designer.cs

```diff
@@ -48,21 +50,26 @@ private void InitializeComponent()
             this.flowLayoutPanel1.Controls.Add(this.Icon);
             this.flowLayoutPanel1.Controls.Add(this._NO_TRANSLATE_Title);
             this.flowLayoutPanel1.Controls.Add(this._NO_TRANSLATE_BranchName);
-            this.flowLayoutPanel1.Location = new System.Drawing.Point(0, 2);
+            this.flowLayoutPanel1.Location = new System.Drawing.Point(4, 4);
             this.flowLayoutPanel1.Margin = new System.Windows.Forms.Padding(0);
             this.flowLayoutPanel1.Name = "flowLayoutPanel1";
             this.flowLayoutPanel1.Size = new System.Drawing.Size(177, 22);
             this.flowLayoutPanel1.TabIndex = 7;
             this.flowLayoutPanel1.WrapContents = false;
+            this.flowLayoutPanel1.MouseEnter += new System.EventHandler(this.DashboardItem_MouseEnter);
```

Comment:
No much point. Still used in several controls and related to DashboardItem in any way. Just DashboardItem itself doesn't have implemented MouseEnter and MouseLeave. Hence hovering was only working while moving mouse over the label only.